setup.py

```diff
@@ -50,6 +50,7 @@
 # =============================================================================
 NAME = 'spyder'
 LIBNAME = 'spyder'
+WINDOWS_INSTALLER_NAME = os.environ.get('EXE_NAME')
```

Comment:
```suggestion
WINDOWS_INSTALLER_NAME = os.environ.get('EXE_NAME')

```

Just add an extra line here so that constants and imports are separated here.